docs/can-view-import.md

```diff
@@ -29,6 +29,52 @@ Dynamically import *MODULE_NAME*; the scope within the template is a [Promise](h
 
 @param {moduleName} [MODULE_NAME] A module that this template depends on.
 
+@signature `<can-dynamic-import from="MODULE_NAME" value:to="*MODULE_REF"/>`
```

Comment:
what's the use case? Should we document?

Reply:
Partials is one use-case.

Reply:
Also when using two or more imports (i.e. components) or two or more uses for each import, if you want to late-load them, this cuts down on nesting.